scss/components/_emergency-banner.scss

```diff
@@ -1,176 +1,135 @@
+// Colours are from gov.uk and not our usual design system choices
+$gov_black: #0b0c0c;
+$gov_red: #b10e1e;
+$gov_teal: #28a197;
+
+.page-type--homepage .emergency-banner {
+  padding: 27px 0;
+}
+
 .emergency-banner {
-    font-family: "Open Sans",Helvetica,Arial,sans-serif;
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-    font-weight: 400;
-    font-size: 16px;
-    line-height: 1.25;
-    background-color: #b1b4b6;
-    color: #fff;
-    margin-top: 2px;
-    padding: 15px 0
-}
-@media print {
-    .emergency-banner {
-        font-family: sans-serif
-    }
-}
-@media (min-width: 40.0625em) {
-    .emergency-banner {
-        font-size:19px;
-        line-height: 1.3157894737;
-    }
-}
-@media print {
-    .emergency-banner {
-        font-size: 14pt;
-        line-height: 1.15
-    }
-}
-@media (min-width: 40.0625em) {
-    .emergency-banner {
-        padding:30px 0
-    }
-}
-.emergency-banner {
-    border-bottom: 5px solid #fff;
-    border-top: 5px solid #fff;
-    margin-bottom: -10px;
-    margin-top: 0;
-    position: relative;
-    z-index: 10
-}
-.emergency-banner__heading {
-    font-family: "nta",arial,sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 1.25;
+  background-color: #b1b4b6;
+  color: $white;
+  margin-top: 2px;
+  padding: 18px 0;
+  border-bottom: 5px solid $white;
+  margin-bottom: -10px;
+  margin-top: 0;
+  position: relative;
+
+  &__heading {
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     font-weight: 700;
-    font-size: 32px;
-    line-height: 1.1111111111;
+    font-size: 30px;
+    line-height: 42px;
```

Comment:
this is dangerous in case of wrapping every line will have lots of space around, use padding

Reply:
That's intentional 
![image](https://user-images.githubusercontent.com/19624419/147089341-3cac572c-2c2f-4bdc-b23b-39de40b94aa8.png)